File: 2025-10/2025-10-04/stellarClassification.js
function classification(temp) {
  let stellar_classification = "";
  
  if (temp >= 30000)
    stellar_classification = "O";
  
  else if (temp >= 10000)
    stellar_classification = "B";
  
  else if (temp >= 7500)
    stellar_classification = "A";
  
  else if (temp >= 6000)
    stellar_classification = "F";
  
  else if (temp >= 5200)
    stellar_classification = "G";
  
  else if (temp >= 3700)
    stellar_classification = "K";
  
  else if (temp >= 0)
    stellar_classification = "M";
  
  return stellar_classification;
}

console.log(classification(5778));
console.log(classification(2400));
console.log(classification(9999));
console.log(classification(3700));
console.log(classification(3699));
console.log(classification(210000));
console.log(classification(6000));
console.log(classification(11432));

File: 2025-10/2025-10-04/stellar_classification.py
def classification(temp):
    
    if temp >= 30000:
        stellar_classification = "O"

    elif temp >= 10000:
        stellar_classification = "B"

    elif temp >= 7500:
        stellar_classification = "A"

    elif temp >= 6000:
        stellar_classification = "F"

    elif temp >= 5200:
        stellar_classification = "G"

    elif temp >= 3700:
        stellar_classification = "K"

    elif temp >= 0:
        stellar_classification = "M"

    return stellar_classification

print(classification(5778))
print(classification(2400))
print(classification(9999))
print(classification(3700))
print(classification(3699))
print(classification(210000))
print(classification(6000))
print(classification(11432))